pkg/monitoring/source_envelope.go

```diff
@@ -122,33 +116,35 @@ func (s *terraSource) Fetch(ctx context.Context) (interface{}, error) {
 			envelopeErr = multierr.Combine(envelopeErr, fmt.Errorf("failed to unmarshal balance response: %w", err))
 			return
 		}
-		balance, err := strconv.ParseUint(balanceRes.Balance, 10, 64)
-		if err != nil {
-			envelopeErr = multierr.Combine(envelopeErr, fmt.Errorf("failed to parse uint64 balance from '%s': %w", balanceRes.Balance, err))
+		balance, success := new(big.Int).SetString(balanceRes.Balance, 10)
+		if !success {
+			envelopeErr = multierr.Combine(fmt.Errorf("failed to parse link balance from '%s'", balanceRes.Balance))
 			return
 		}
-		envelope.LinkBalance = balance
+		envelope.LinkBalance = uint64(balance.Int64())
```

Comment:
Why the switch from parsing directly as uint64? Should this conversion be checked first with `IsInt64()`, since it's undefined otherwise? (And/or could `Uint64()/IsUint64()` be used?)

Reply:
I had a convo with @kaleofduty and these balances - as well as the observations - will more than fit in a float64.

Reply:
So going forward, we're going to read big.Ints from the chain; convert them to int64 - without the IsInt64 check - and then from there to a float64 ready for export to prometheus.